Automattic/wp-calypso
# Vertical Settings Control: a new control to add semantic tags to any block

#### Changes proposed in this Pull Request

* Implement a new vertical settings control and is only enabled on the site with either “block-patterns-source-site“ or ”theme-demo-site“ sticker
* This control has a semantic tags field and it would fetch the vertical terms and show them as suggestions
* After you add any tags, they would be stored in `verticalSettings.semanticTags` of the block attributes

https://user-images.githubusercontent.com/13596067/161242730-6d3066c1-d85b-42fc-af97-034a8682d7ad.mov

#### Testing instructions



* Ensure D78021-code is deployed
* Run `cd apps/editing-toolkit && yarn dev --sync`
* Go to specified sites
* Create or edit any post
* Select any block and go to block settings
* You could see “Vertical Settings” control in block settings and you could add the semantic tags
* Add the semantic tags, save the post, and refresh the page. Ensure that the semantic tags for that block still exist



Related to https://github.com/Automattic/wp-calypso/issues/62197
